frontend/src/lib/components/AddInsightsToDashboard/AddInsightsToDashboard.tsx

```diff
@@ -0,0 +1,49 @@
+import { LemonButton } from '@posthog/lemon-ui'
+import { useActions } from 'kea'
+import { router } from 'kea-router'
+import { urls } from 'scenes/urls'
+
+interface AddInsightsToDashboardProps {
+    dashboardId: number
+    setAddInsightsToDashboardModalOpen: (open: boolean) => void
+    disabledReason: string | null
+}
+
+export function AddInsightsToDashboard({
```

Comment:
```suggestion
export function AddToDashboardButton({
```

Reply:
Both buttons that were replaces with this component had a `data-attr="..."`. We use these for analytics tracking and e2e tests (this might explain the failing cypress tests, didn't look).

One button was also centered before.